frontend/src/views/AccountView.vue

```diff
@@ -42,49 +43,52 @@ export default {
   components: {
     RosalutionHeader,
     SectionBox,
+    CredentialsBox,
   },
   data() {
     return {
       showSecretValue: false,
+      clientSecret: '',
     };
   },
+  created() {
+    this.clientSecret = this.user.clientSecret;
+  },
   computed: {
     user() {
       return authStore.getUser();
     },
-    credentialsContent() {
-      const content = [
-        {field: 'Client ID', value: [this.user.clientId]},
-        {
-          field: 'Client Secret',
-          value: this.secretValue,
-        },
-      ];
-      return content;
-    },
     secretValue() {
       if (this.showSecretValue) {
-        return [this.user.clientSecret];
+        return this.clientSecret;
       }
-      if (this.user.clientSecret) {
-        return ['<click to show>'];
+      if (this.clientSecret) {
+        return '<click to show>';
       }
 
-      return ['<empty>'];
+      return '<empty>';
+    },
+    clientSecretExists() {
+      return !!this.clientSecret;
```

Comment:
What is the "!!" not operator intended for?  A quick google didn't yield anything. Curious where you found this or if it is an error

Reply:
Thats interesting.  What article did you find this in? I want to read more about it. Will resolve it and mention it in slack.